ConsoleApp/Point.cs

```diff
@@ -0,0 +1,69 @@
+using System;
+
+namespace Snake
+{
+    class Point
+    {
+        public int x;
```

Comment:
code convention violated here
```suggestion
        public int X;
```